tests/io/test_metar.py

```diff
@@ -163,6 +163,14 @@
      Metar('TFFF', 14.6, -61.0, 5, datetime(2017, 5, 11, 18, 30), 110, 19, 1000, 'RA', 'BCFG',
            np.nan, 'FEW', 1400, 'BKN', 2100, 'BKN', 2700, np.nan, np.nan, 6, 27, 24,
            units.Quantity(1015, 'hPa').m_as('inHg'), 63, 41, 0, '')),
+    # Interchanged wind and vis groups
+    ('KBMI 121456Z COR 10SM 10005KT SCT055 OVC065 23/22 A3000 RMK AO2'
```

Comment:
A missing space here is causing the test to fail:
```suggestion
    ('KBMI 121456Z COR 10SM 10005KT SCT055 OVC065 23/22 A3000 RMK AO2 '
```

Reply:
Ahh, I knew there was something small like this but couldn't find it. Thanks.

However, the test is still failing on my local machine after that change, so there must be something else amiss as well. I'll see if I can find it.